testdata/well_known/convert_well_known.sh

```diff
@@ -2,7 +2,7 @@ ACCOUNT_NAME=
 SAS=
 
 python make_well_known.py well_known.segy
-SEGYImport --url "file://." --vdsfile well_known_default.vds well_known.segy
+SEGYImport --url "file://." --vdsfile well_known_default.vds well_known.segy --crs-wkt="utmXX"
```

Comment:
It's one of these thinks that doesn't have a defined entry anywhere in the SEGY, as a result it's written in the text header. Because it's free text, it must be added "manually".  In practice it's almost always there, because knowing the coordinate reference system is pretty much a minimum requirement if you want to work with coordinates.